modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -219,6 +219,9 @@ public void processOpts() {
             importMapping.remove("DateTime");
             instantiationTypes.put("array", "java.util.ArrayList");
             instantiationTypes.put("map", "java.util.HashMap");
+        } else {
+            languageSpecificPrimitives.add("List");
+            languageSpecificPrimitives.add("Map");
```

Comment:
For other languages, we do add List (array) and Map (dictionary, hash) to the languageSpecificPrimitives (which means no additional import of package/modules is required) but for Java, it requires explicit import of `java.util.List` and `java.util.Map`.

Are you trying to improve the Java import to remove unnecessary import statements from the Java API client?